include/sway/input/text_input_popup.h

```diff
@@ -9,6 +9,7 @@ struct sway_input_popup {
 	struct wlr_scene_tree *scene_tree;
 	struct sway_popup_desc desc;
 	struct wlr_input_popup_surface_v2 *popup_surface;
+	struct wlr_output *fixed_output;
```

Comment:
Took me a bit to figure out that the popup is tied to the layer surface which is tied to the output, so it's OK to store an output pointer without a destroy listener. Maybe we could add a small comment to make this clear.

(Maybe it would be a tad better to fish back the layer surface when needed instead of adding more state?)